src/py/mat3ra/made/tools/modify.py

```diff
@@ -35,22 +34,46 @@ def filter_by_label(material: Material, label: Union[int, str]) -> Material:
     return new_material
 
 
-@decorator_convert_material_args_kwargs_to_structure
-def translate_to_bottom(structure: PymatgenStructure, use_conventional_cell: bool = True):
+def translate_to_z_level(
+    material: Material, z_level: Optional[Literal["top", "bottom", "center"]] = "bottom"
+) -> Material:
     """
-    Translate atoms to the bottom of the cell (vacuum on top) to allow for the correct consecutive interface generation.
-    If use_conventional_cell is passed, conventional cell is used.
+    Translate atoms to the specified z-level.
 
     Args:
-        structure (Structure): The pymatgen Structure object to normalize.
-        use_conventional_cell: Whether to convert to the conventional cell.
+        material (Material): The material object to normalize.
+        z_level (str): The z-level to translate the atoms to (top, bottom, center)
     Returns:
-        Structure: The normalized pymatgen Structure object.
+        Material: The translated material object.
     """
-    if use_conventional_cell:
-        structure = PymatgenSpacegroupAnalyzer(structure).get_conventional_standard_structure()
-    structure = translate_to_bottom_pymatgen_structure(structure)
-    return structure
+    atoms = to_ase(material)
+    min_z = min(atoms.positions[:, 2])
```

Comment:
Expressions like `[:,2]` are hard to read, I suggest we define helpers

```python
min_z = (from ..analyze ) get_atomic_coordinates_min_z(material)
max_z = ...
then instead of external atoms.translate use `traslate_by_vector([0, 0, 1-max_z]) etc.

This way we isolate the use of atoms to a single place
```